### PR TITLE
Build fixes (arm, minimal)

### PR DESCRIPTION
**Pull request by proller**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Build/Testing/Packaging Improvement

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):
Build fixes (arm, minimal)

### CODE REVIEW

File: dbms/src/Processors/Formats/Impl/CapnProtoRowInputFormat.cpp [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #include "CapnProtoRowInputFormat.h"
-#if USE_CAPNP
+#include <Formats/FormatFactory.h>
```
Comment (alexey-milovidov) [inline dbms/src/Processors/Formats/Impl/CapnProtoRowInputFormat.cpp]: I see no point for this change.

Comment (proller) [reply dbms/src/Processors/Formats/Impl/CapnProtoRowInputFormat.cpp]: All changes about moving formatfactory fixes error like this:
```
FAILED: dbms/CMakeFiles/dbms.dir/src/Storages/StorageXDBC.cpp.o
/usr/lib/ccache/g++-8   -isystem ../contrib/sparsehash-c11 -isystem ../contrib/libdivide -isystem ../contrib/pdqsort -Idbms/includes/configs -Idbms/src/Core/include -isystem ../contrib/libpcg-random/include -I..
/dbms/src -Idbms/src -Idbms/src/Formats/include -I../libs/libcommon/include -Ilibs/libcommon/include -I../contrib/cityhash102/include -isystem ../contrib/croaring -isystem ../contrib/libbtrie/include -fdiagnosti
cs-color=always -fsized-deallocation  -pipe -msse4.1 -msse4.2 -mpopcnt   -Wall  -Wnon-virtual-dtor -Wextra -Werror -O2 -g -DNDEBUG -O3  -fPIC   -D OS_LINUX -fno-tree-loop-distribute-patterns -Wbool-compare -Wcast-align -Wcast-qual -Wdelete-incomplete -Wdisabled-optimization -Wduplicated-cond -Wenum-compare -Winit-self -Wlogical-not-parentheses -Wlogical-op -Wmaybe-uninitialized -Wmisleading-indentation -Wmissing-declarations -Wnon-virtual-dtor -Wno-return-local-addr -Wnull-dereference -Wodr -Wold-style-cast -Wplacement-new=2 -Wpointer-arith -Wredundant-decls -Wreorder -Wshadow -Wshift-negative-value -Wsized-deallocation -Wsiz
eof-array-argument -Wsizeof-pointer-memaccess -Wswitch-bool -Wtautological-compare -Wtrampolines -Wunused -Wvector-operation-performance -std=c++1z -MD -MT dbms/CMakeFiles/dbms.dir/src/Storages/StorageXDBC.cpp.o -MF dbms/CMakeFiles/dbms.dir/src/Storages/StorageXDBC.cpp.o.d -o dbms/CMakeFiles/dbms.dir/src/Storages/StorageXDBC.cpp.o -c ../dbms/src/Storages/StorageXDBC.cpp
../dbms/src/Storages/StorageXDBC.cpp:139:6: error: no previous declaration for 'void DB::registerStorageODBC(DB::StorageFactory&)' [-Werror=missing-declarations]
 void registerStorageODBC(StorageFactory & factory)
      ^~~~~~~~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
[
```

Comment (amosbird) [reply dbms/src/Processors/Formats/Impl/CapnProtoRowInputFormat.cpp]: alternatively we can disable that warning https://github.com/ClickHouse/ClickHouse/pull/8281

Comment (alexey-milovidov) [reply dbms/src/Processors/Formats/Impl/CapnProtoRowInputFormat.cpp]: Ok. Now we can update this PR.

**Review by alexey-milovidov — APPROVED** [review]

**Pull request merged**